src/app/scripts/cac/control/cac-control-tour-list.js

```diff
@@ -0,0 +1,184 @@
+        // Should the share button be shown in the control
+        showShareButton: false,
+        selectors: {
+            alert: '.alert',
+            container: '.directions-list',
+            hiddenClass: 'hidden',
+            destinationList: '.tour-list',
+            destinationItem: '.tour-place-card',
+            destinationDirectionsButton: '.tour-place-action-directions'
+        }
+    };
+    var options = {};
+
+    var events = $({});
+    var eventNames = {
+        destinationClicked: 'cac:control:tourlist:destinationclicked',
+        destinationHovered: 'cac:control:tourlist:destinationhovered'
+    };
+
+    var $container = null;
+    var destinations = [];
+
+    function TourListControl(params) {
+        options = $.extend({}, defaults, params);
+        $container = $(options.selectors.container);
+    }
+
+    TourListControl.prototype = {
+        events: events,
+        eventNames: eventNames,
+        setTourDestinations: setTourDestinations,
+        setTourDestinationsError: setTourDestinationsError,
+        showTourDestinations: showTourDestinations,
+        show: show,
+        hide: hide,
+        toggle: toggle
+    };
+
+    return TourListControl;
+
+    function setTourDestinations(newDestinations, tourName) {
+        destinations = newDestinations;
+
+        // Show the directions div and populate with tour destinations
+        var html = MapTemplates.tourDestinationList(destinations, tourName);
+        $container.html(html);
+
+        // TODO: mobile view
+        //enableCarousel(destinations);
+
+        $(options.selectors.destinationDirectionsButton).on('click', onTourDestinationClicked);
+        $(options.selectors.destinationItem).hover(onTourDestinationHovered);
+    }
+
+    /**
+     * Use in case OTP planner returns an error
+     * @param {Object} Error object returned from OTP
+     */
+    function setTourDestinationsError(error) {
+
+        // default message; will use in case OTP unresponsive
+        var msg = 'Cannot currently plan trip. Please try again later.';
+
+        // If OTP responded with an error, use its message
+        if (error && error.msg) {
+            msg = error.msg;
+            // override default error message for out-of-bounds or non-navigable orign/destination
+            if (msg.indexOf('Your start or end point might not be safely accessible') > -1) {
+                msg = 'Make sure the origin and destination are accessible addresses within ' +
+                      'the Greater Philadelphia area.';
+            }
+        }
+
+        var alert = MapTemplates.alert(msg, 'Could not plan trip', 'danger');
+        $container.html(alert);
+
+        // handle alert close button click
+        $container.one('click', options.selectors.alert, function() {
+            $(options.selectors.alert).remove();
+        });
+    }
+
+    /**
+     * Enable carousel for swiping destinations on mobile
+     */
+    function enableCarousel(destinations) {
```

Comment:
It might be nicer to put this in when it's activated rather than committing it unused now, to make #1147 more self-contained.